### DISCUSSION TITLE
no access point in server farm in access server.

### DISCUSSION THREAD

**Discussion by farzinsb** [vpnhood/VpnHood#412] [Q&A]
hi, I get authentication error on some servers. when I checked the access server farm , it shows it has no access points!
autoconfiguration is on   in the server  and its ubuntu. servers updated to version432 and this happened.

Access Point Type

IP Addresses in token (Fake Domain)
Access Points

(Nothing)

**Comment by farzinsb** [comment]
{
  "Data": {},
  "TypeName": "InvalidOperationException",
  "TypeFullName": "System.InvalidOperationException",
  "Message": "Could not find any public access point for the ServerFarm. Please configure a server for this AccessToken."
}

**Comment by trudyhood** [comment]
duplicate of #411
